android/app/src/main/java/org/unicef/ecar/bebbo/ExactAlarmModule.java

```diff
@@ -0,0 +1,57 @@
+package org.unicef.ecar.bebbo;
+
+import android.app.AlarmManager;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.provider.Settings;
+import androidx.annotation.NonNull;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+
+public class ExactAlarmModule extends ReactContextBaseJavaModule {
+
+    private final ReactApplicationContext reactContext;
+
+    ExactAlarmModule(ReactApplicationContext context) {
+        super(context);
+        this.reactContext = context;
+    }
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "ExactAlarmModule";
+    }
+
+    @ReactMethod
+    public void requestExactAlarmPermission(Promise promise) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {  // Android 13 and above
+            AlarmManager alarmManager = (AlarmManager) reactContext.getSystemService(Context.ALARM_SERVICE);
```

Comment:
Add try/catch to handle errors when system fails or AlarmManager is not available.